lua/lspconfig/server_configurations/daedalus_ls.lua

```diff
@@ -0,0 +1,33 @@
+local util = require 'lspconfig.util'
+local bin_name = 'DaedalusLanguageServer'
+
+if vim.fn.has 'win32' == 1 then
+  bin_name = bin_name + '.exe'
+end
```

Comment:
```suggestion

```

Reply:
also, if this line were kept, the operator is wrong (should be `..`)

Reply:
May I ask why should I remove it? I've checked some servers configs in repo and they had that. Or is it automatic now?

Reply:
which servers ?

Reply:
oh I see. they are also should be removed i will do it later. we use exepath to find server executable so no need it now

Reply:
Done.